### PR TITLE
Improve KO chance 

### PR DESCRIPTION
**Pull request by AustinXII**
this is a WIP I'm going to add all the relevant moves, but yay!

### CODE REVIEW

**Comment by AustinXII** [comment]
I want to make sure this is what you meant, @Marty-D it does work after testing

**Comment by AustinXII** [comment]
also fyi, fixed ko chance of stat dropping moves after this PR :)

File: js/ko_chance.js [hunk 1]
```diff
@@ -136,7 +136,18 @@
...
+    if (move.name === 'Bind' || move.name === 'Clamp' || move.name === 'Fire Spin' || move.name === 'Infestation' || move.name === 'Sand Tomb' || move.name === 'Whirlpool' || move.name === 'Wrap') {
+        eot -= Math.floor(defender.maxHP / 8);
+        eotText.push('trapping damage');
+    }
+    if ((move.name === 'Fire Pledge (Grass Pledge Boosted)' || move.name === 'Grass Pledge (Fire Pledge Boosted)') && (defender.type1 !== "Fire" && defender.type2 !== "Fire")) {
+        eot -= Math.floor(defender.maxHP / 8);
+        eotText.push('Sea of Fire damage');
+    }
+    if (move.name === 'Magma Storm') {
+        eot -= gen > 4 ? Math.floor(defender.maxHP / 8) : Math.floor(defender.maxHP / 16);
```
Comment (Marty-D) [inline js/ko_chance.js]: `gen > 5`
Also Magma Storm itself can move into the Bind section.

**Comment by AustinXII** [comment]
This is mergable if you don't see any other issues

**Pull request merged**